source/dlangbot/bugzilla.d

```diff
@@ -27,16 +27,27 @@ auto matchIssueRefs(string message)
     }
 
     // see https://github.com/github/github-services/blob/2e886f407696261bd5adfc99b16d36d5e7b50241/lib/services/bugzilla.rb#L155
-    enum issueRE = ctRegex!(`((close|fix|address)e?(s|d)? )?(ticket|bug|tracker item|issue)s?:? *([\d ,\+&#and]+)`, "i");
+    enum issueRE = ctRegex!(`((fix)(es)? )?(bug|issue)s?:? *([\d ,\+&#and]+)`, "i");
```

Comment:
```suggestion
    enum issueRE = ctRegex!(`(fix(es)? )?(bug|issue)s?:? *([\d ,\+&#and]+)`, "i");
```

Reply:
Just wanted to ease my life by keeping the same number of capture, but it's failing anyway, so will fix.

Reply:
Fair enough